solution.cpp

```diff
@@ -11,11 +11,150 @@
 
 // ----------------------------------------------------------------------------
 
-int main() {
-    // TODO
+/* ----------------------------------------------------------------------------
+* Copyright &copy; 2015 David Ferranco <dgferranco@hotmail.com>
+* Released under the [MIT License] (http://opensource.org/licenses/MIT)
+* ------------------------------------------------------------------------- */
+
+#include "stdafx.h"
```

Comment:
i've probably mentioned this to you in lab by now, but this is a windows only thing -- best to avoid it, unless for some reason it's necessary :-)